serialization/json/README.md

```diff
@@ -0,0 +1,87 @@
+# JSON Serialized Examples
+
+[XML/RDF](https://cambridgesemantics.com/blog/semantic-university/learn-rdf/rdf-vs-xml/#:~:text=This%20brings%20us%20to%20the,is%20primarily%20a%20data%20model.),
+Turtle, and JSON-LD are special-purpose data formats used to serialize
+[RDF graphs](https://www.w3.org/TR/owl2-overview/#Overview).
+In contrast, XML and JSON are general purpose data formats used to serialize arbitrary data,
+including non-RDF applications such as
+[GPX](https://en.wikipedia.org/wiki/GPS_Exchange_Format) or
+[XLSX](https://learn.microsoft.com/en-us/openspecs/office_standards/ms-xlsx/f780b2d6-8252-4074-9fe3-5d7bc4830968).
+Other general data formats include YAML, tag-value, and spreadsheet, as well as machine-oriented
+formats such as [CBOR](https://cbor.io/), [Protobuf](https://protobuf.dev/),
+and [Avro](https://avro.apache.org/).
+
+JSON and other general data formats do not:
+* assume that all data instances are nodes in a graph,
+* reserve property names (such as @context, @id, @type, @graph) for special purposes, or
+* require applications to perform any special processing based on reserved property names.
+
+### JSON-LD
+A collection of examples that is "almost JSON except with @context" is a JSON-LD serialization, not JSON.
+JSON serialization does not use JSON-LD processing or features or sandbox capabilities.
+But RDF-aware applications can deserialize pure JSON data because the model defines the data types
+and their RDF links.
+
+For example, a JSON message:
+```json
+[38.88949, -77.03529]
+```
+is complete in and of itself.
+* Naive applications can parse it into an array of two numbers.
+* Model-aware applications know the array type, its two properties, and those property
+types because they are defined by the model.
+* RDF-aware applications know the array and property types and the RDF URIs of all those types,
+because the model includes those URIs:
+```
+Coordinate = Array       // rdf="http://geotags.org/GPS-Coordinate"
+  1 latitude Latitude
+  2 longitude Longitude
+
+Latitude = Number{-90.0..90.0}   // rdf="http://geotags.org/Latitude"
+```
+There is no need to transmit RDF information in JSON-serialized data, and there is no reason to
+require all JSON-consuming applications to recognize RDF information, even if only to discard it.
+
+### Definitions:
+* **Logical Value** - the value of an application variable prior to serializing or
+after deserializing an Element instance.
+* **ElementSet** - an arbitrary set of Logical Values.
+* **Payload** - the result of serializing an ElementSet.
+* **SpdxDocument** - an SPDX Element containing metadata about a Payload.
```

Comment:
your definitions do not align with https://github.com/spdx/spdx-3-model/blob/main/model/Core/Classes/SpdxDocument.md. Instead of discussing it here buried down in a long file in some PR, maybe you can create issues and PRs for such statements.

Reply:
They do agree, but the distinction between the logical model (which the diagram and model classes represent) and the physical model (which is represented by the payload <<interface>> and which we are just getting around to creating now) is still not clear.

A physical File is a bunch of C code or anything else that goes in files.  A physical Package is something that you install from a package manager.  A physical SpdxDocument is the serialized value of a bunch of Elements.

* An SPDX File logical element as contained in the model doesn't contain C code, it describes the C code.
* An SPDX Package logical element cannot be installed from a package manager, it describes the package that can be installed.
* An SPDX SpdxDocument logical element does not contain serialized SPDX Element values, it describes serialized bytes of those values.  There is only one SpdxDocument logical element that describes a dozen physical SpdxDocument serialized byte strings.

Because the logical model isn't a physical model, we had long discussions about how to represent a series of bytes on the logical model diagram and the logical model files, and William came up with the name "Payload" for the physical SpdxDocument to distinguish it from a logical Element.